templates/init.pp

```diff
@@ -24,6 +24,11 @@
   # udp = faster but many firewalls block it
   proto        => '{{protocol}}',
   port         => '{{port}}',
+  # Fix for pushing routes to the address space, but not the gateway
+  {%- if not push_default_gateway %}
+  push         => ['{{network}}/24'],
```

Comment:
@galgalesh I'm pretty sure this will do the trick. How can we get the net bit mask of the private address space dynamically?